frontend/src/metabase/visualizations/click-actions/actions/CombineColumns/CombineColumns.tsx

```diff
@@ -0,0 +1,58 @@
+import { t } from "ttag";
```

Comment:
Column info popups don't work well:

https://github.com/metabase/metabase/assets/6830683/ebbdd27f-937a-46f3-a8d6-12c4d522794e

Reply:
There's actually another project I'm working on that will fix the weird behavior of these popovers!